playbooks/tasks/iworx-php-scl.yml

```diff
@@ -7,12 +7,21 @@
     -c Http
     -a enableMultiplePhp
 
+- name: Find Installed PHP Versions
+  find:
+    paths: "/opt/remi/"
```

Comment:
/opt/remi should be defined in nexcess-role-php and then referenced here.

Reply:
It looks like you're relying on 'find' to return things in alpha order.  You may want to do a 'command' call that does 'ls' that forces an order just to make sure it's idempotent and doesn't fail from some weird 'find' sorting issue.